pkg/sloop/webserver/debug.go

```diff
@@ -162,6 +164,116 @@ func listKeysHandler(tables typed.Tables) http.HandlerFunc {
 	}
 }
 
+type sloopKeyInfo struct {
+	MinimumSize int64
+	MaximumSize int64
+	TotalKeys   int64
+	TotalSize   int64
+	AverageSize int64
+}
+
+type sloopKey struct {
+	TableName   string
+	PartitionID string
+}
+
+type histogram struct {
+	HistogramMap map[sloopKey]*sloopKeyInfo
+	TotalKeys    int
+	DeletedKeys  int
+}
+
+// returns TableName, PartitionId, error.
+func parseSloopKey(item badgerwrap.Item) (string, string, error) {
```

Comment:
this method could be refactored to return `(sloopKey, error)`

Reply:
Yes but I want to keep table name and partition in their separate columns. It helps in better analysis of data.